zero/gkyl_mirror_geo.h

```diff
@@ -80,6 +89,9 @@ struct gkyl_mirror_geo_grid_inp {
   plate_func plate_func_lower;
   plate_func plate_func_upper;
 
+  bool exact_roots; // If true we will allow approximate roots when no root is found
```

Comment:
This feels like the name of this variable should be approx_roots, not exact_roots, given the comment.

Reply:
The comment is wrong, I'll fix this